### DISCUSSION TITLE
安装了nvm 之后在安装全局 yarn 问题如图

### DISCUSSION THREAD

**Discussion by xinghaodong** [1111mp/nvm-desktop#141] [Q&A]
![Uploading 微信图片_20241116224113.png…]()

**Comment by xinghaodong** [comment]
图片打不开 
终端 提示 nvm-desktop: command not found: "yarn"

**Comment by 1111mp** [comment]
你看看这里的文档能否帮助你解决问题呢 https://github.com/1111mp/nvm-desktop/issues/23
